src/util/MongoBinaryDownload.js

```diff
@@ -18,22 +18,40 @@ export type MongoBinaryDownloadOpts = {
   platform: string,
   arch: string,
   debug?: DebugPropT,
+  skipMD5?: boolean,
 };
 
 export default class MongoBinaryDownload {
   debug: DebugFn;
   dlProgress: DownloadProgressT;
 
+  skipMD5: boolean;
   downloadDir: string;
   arch: string;
   version: string;
   platform: string;
 
-  constructor({ platform, arch, downloadDir, version, debug }: $Shape<MongoBinaryDownloadOpts>) {
+  constructor({
+    platform,
+    arch,
+    downloadDir,
+    version,
+    skipMD5,
+    debug,
+  }: $Shape<MongoBinaryDownloadOpts>) {
     this.platform = platform || os.platform();
     this.arch = arch || os.arch();
     this.version = version || 'latest';
     this.downloadDir = path.resolve(downloadDir || 'mongodb-download');
+    if (skipMD5 === undefined) {
+      this.skipMD5 =
```

Comment:
Just to be consistent with a recent merged PR (https://github.com/nodkz/mongodb-memory-server/pull/110/files#diff-551007241ae61889b0f85f47aa764343R16), could you use this syntax instead : 
```javascript
this.skipMD5 =(typeof process.env. MONGOMS_SKIP_MD5_CHECK === 'string') &&
  (['1', 'on', 'yes', 'true'].indexOf(process.env. MONGOMS_SKIP_MD5_CHECK.toLowerCase()) !== -1)
```

Reply:
@AJRdev updated but without parentheses. Eslint removes parentheses in the expression.